homeassistant/components/websocket_api/commands.py

```diff
@@ -138,13 +138,20 @@ async def handle_call_service(hass, connection, msg):
 
```

Comment:
Also enable support for templates in target, the line above this comment.

Reply:
Is it enough to remove the check? If so, fixed by 7136020